### PR TITLE
Simple Signup Description change - Username Lowercases

### PR DESCRIPTION
**Pull request by Techguy9078**
- Just doing a little edit for usability

### CODE REVIEW

File: src/pages/auth/signup.tsx [hunk 1]
```diff
@@ -26,225 +26,225 @@ import { zodResolver } from "@hookform/resolvers/zod";
...
+		.object({
+			lastName: z.string({ required_error: "Please enter your last name" }),
+			firstName: z.string({ required_error: "Please enter your first name" }),
+
+			username: z
+				.string()
+				.min(3, { message: "Username must contain at least 3 characters" })
+				.max(15, { message: "Username cannot contain more than 15 characters" })
+				.regex(/^[a-z0-9_-]{3,15}$/gm, {
+					message: "Please enter a valid username, only lowercases",
```
Comment (mikayelgr) [inline src/pages/auth/signup.tsx]: This is the only change, right? Could you please change `lowercases` to `lowercase`?

**Comment by mikayelgr** [comment]
Also, could you revert the formatting back to its initial state? Thank you.

**Comment by Techguy9078** [comment]
I was wondering why it had so many changes dammit.
I'm guessing this would be due to extension, prettier haha.

**Comment by mikayelgr** [comment]
> I was wondering why it had so many changes dammit. I'm guessing this would be due to extension, prettier haha.

Yeah :smile:, nevermind my comments, I have updated the PR. Let's just wait for the preview to build and I will merge it right away.

**Review by mikayelgr — APPROVED** [review]

**Comment by Techguy9078** [comment]
Can we put a format file in the main directory for text-formatters

**Review by Techguy9078 — COMMENTED** [review]
Yes this was the change

**Comment by mikayelgr** [comment]
> Can we put a format file in the main directory for text-formatters

Yup, I would make sure to add it to the root of the project. Thanks!

**Comment by Techguy9078** [comment]
> > Can we put a format file in the main directory for text-formatters
> 
> Yup, I would make sure to add it to the root of the project. Thanks!

Thankyou, would help to stop that next time :)

**Comment by mikayelgr** [comment]
LGTM! Thanks for your contribution!

**Pull request merged**